app/controllers/response_controller.rb

```diff
@@ -211,13 +217,21 @@ def create
     redirect_to :controller => 'response', :action => 'saving', :id => @map.map_id, :return => params[:return], :msg => msg, :error_msg => error_msg, :save_options => params[:save_options]
   end
 
+  #E1600
+  #Added paramps[:return] value for 'SelfReviewResponseMap'
   def saving
     @map = ResponseMap.find(params[:id])
+    if(@map.type == "SelfReviewResponseMap")
```

Comment:
Is there any way that you can make params[:return] equals "selfreview" from view? Making this change in the controller does not seem to be a good idea.